test/exampleTestHelpers.js

```diff
@@ -0,0 +1,147 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
```

Comment:
- why is this here? It should go in the examples foler.
- you also need to add a github workflow to run the tests for all the example apps on each PR creation (see how it's done in auth-react repo)

Reply:
> why is this here? It should go in the examples foler.
This is how it is auth-react repo